src/components/data/data-utils.js

```diff
@@ -145,20 +145,35 @@ export const renderDataCell = (attributeValue, workspace) => {
 
   const tooltip = renderDataCellTooltip(attributeValue)
 
-  const isOtherBucketGsUri = datum => {
-    const [bucket] = parseGsUri(datum)
-    return !!bucket && bucket !== workspaceBucket
+  const isNonCurrentWorkspaceUrls = datum => {
+    if (isGoogleWorkspace(workspace)) {
+      const [bucket] = parseGsUri(datum)
+      return !!bucket && bucket !== workspaceBucket
+    } else if (isAzureWorkspace(workspace)) {
+      if (isAzureUri(datum)) {
+        const workspaceId = parseAzureUri(datum)
+        return (!!workspaceId && workspaceId !== workspace.workspace.workspaceId)
```

Comment:
Is there any case here in which `!!workspaceId` returns a false negative? maybe just
```javascript
return workspace.workspace.workspaceId !== workspaceId
```
would cover it.

This could just be a code style/preference so feel free to ignore it :)

Reply:
no I think you're right! thanks for the suggestion!